numba/targets/mathimpl.py

```diff
@@ -410,3 +412,65 @@ def degrees_float_impl(context, builder, sig, args):
 def pow_impl(context, builder, sig, args):
     impl = context.get_function(operator.pow, sig)
     return impl(builder, args)
+
+# -----------------------------------------------------------------------------
+
+
+@intrinsic
```

Comment:
Perhaps put this in `numba.unsafe.numbers` ?

Reply:
`trailing_zeros`, `unsigned`, or both?

Reply:
`_trailing_zeros`, it's a useful intrinsic

Reply:
@stuartarchibald, I've added that (along with `leading_zeros`, since they're useful together).

Reply:
@stuartarchibald, I've added that (along with `leading_zeros`, since they're useful together).

Reply:
Brilliant, thanks for doing that!